src/App/Service/Github/Query.php

```diff
@@ -54,7 +55,8 @@ class Query
             . ' -' . self::LABEL_WAITING_FOR_UX
             . ' -' . self::LABEL_WAITING_FOR_WORDING
             . ' -' . self::LABEL_WAITING_FOR_QA
-            . ' -' . self::LABEL_WIP,
+            . ' -' . self::LABEL_WIP
+            . ' -' . self::LABEL_BLOCKED,
```

Comment:
Next time it could be cool to refactor it by using and array and implode it instead of long concatenation string.